POLICIES.md

```diff
@@ -0,0 +1,106 @@
+# Portieris Policies
+
+## Image Policy Resources
+
+Portieris defines two custom resource types for policy:
+
+* ImagePolicies can be configured in each Kubernetes namespace, and define Portieris' behavior in that namespace. If an ImagePolicy exists in a namespace, the policies from that namespace are used, even if the ImagePolicy does not have a matching policy for a given image. If a namespace does not have an ImagePolicy, the ClusterImagePolicy is used.
```

Comment:
Can we clarify and document what happens if an image does not match any of the repository strings. 

So you hit this case "If an ImagePolicy exists in a namespace, the policies from that namespace are used" 

May want to go in the `Repository Matching` section